src/databricks/sql/thrift_backend.py

```diff
@@ -31,6 +34,8 @@
 
 TIMESTAMP_AS_STRING_CONFIG = "spark.thriftserver.arrowBasedRowSet.timestampAsString"
 
+DEFAULT_RETRY_DELAY_AFTER = 5
```

Comment:
nit thought: should this be in the _retry_policy below? It's not bounds, but could be a `_retry_delay_default = (float, 5, None, None)` named entry. Placing it there gives a clear path for the savy/hacky user to set a new default in a blessed, per-connection way.

Reply:
Applied in [2e80e44](https://github.com/databricks/databricks-sql-python/pull/18/commits/2e80e4461822c382dbd40ccca24371daf3506fa4).